project/Testing.scala

```diff
@@ -32,10 +36,36 @@ object Testing {
     }
   }
 
+  val minnieKenny = inputKey[Unit]("Run minnie-kenny.")
+
   def isIntegrationTest(name: String) = name contains "integrationtest"
 
   lazy val IntegrationTest = config("it") extend Test
 
+  /** Run minnie-kenny only once per sbt invocation. */
+  class MinnieKennySingleRunner() {
+    private val mutex = new Object
+    private var resultOption: Option[Int] = None
```

Comment:
I followed Cromwell and Agora pattern to use a var and change resultOption value. Not sure how to solve the Codacy error. Open to any suggestions.

Reply:
ignore the codacy error